packages/styles/scss/components/card-in-card/_card-in-card.scss

```diff
@@ -98,6 +102,7 @@
       opacity: 0;
       z-index: 1;
       background-color: $inverse-02;
+      transition: all 150ms cubic-bezier(0.2, 0, 0.38, 0.9) 0s;
     }
 
     .#{$prefix}--image__img {
```

Comment:
Is there a good way we could apply it to the card image as well? I also noticed we're using different techniques. Card in card uses opacity on a psuedo element while card image uses filter. Just thinking out loud we do something like this?

```scss
/* image component */
[data-autoid*=dds--image] {
    filter: brightness(1);
    transition: opacity $moderate-01 motion('standard', 'productive');
}

/* card component */
.bx--card:hover [data-autoid*=dds--image] {
    filter: brightness(.8);
}
```

Feel free to poke holes, but at a minimum could we fix card with image? It appears to be a similar, if not the same issue.